ios/CalendarsExample.xcodeproj/project.pbxproj

```diff
@@ -683,18 +683,21 @@
 				TargetAttributes = {
 					00E356ED1AD99517003FC87E = {
 						CreatedOnToolsVersion = 6.2;
+						DevelopmentTeam = H9Y8849Q6U;
 						TestTargetID = 13B07F861A680F5B00A75B9A;
 					};
 					13B07F861A680F5B00A75B9A = {
-						DevelopmentTeam = 78G36M9BB8;
+						DevelopmentTeam = H9Y8849Q6U;
```

Comment:
You should revert changes in this file (I will think of a way to not change DevelopmentTeam every time)

Reply:
Recoverd